content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,263 @@
+   - :doc:`Wave picking <../picking_methods/wave>`
+   - :doc:`../picking_methods/cluster`
+
+To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
+reveals a list of outgoing deliveries.
+
+.. _inventory/shipping_receiving/assign-carrier:
+
+Carrier assignment
+------------------
+
+Reveal the *Carrier* column, if it is not visible by default, by clicking the
+:icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the to-right corner, and ticking the
+:guilabel:`Carrier` checkbox.
+
+.. tip::
+   Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight`, and
+   :guilabel:`Shipping Volume`.
+
+Select the delivery orders for the batch by ticking the checkboxes on the left. Next, click into the
+line's :guilabel:`Carrier` fields. In the resulting drop-down menu, choose the desired vehicle's
+:ref:`delivery method <inventory/shipping_receiving/delivery-method-for-carrier>`. A
+:guilabel:`Confirmation` pop-up window appears, indicating the number of orders being added to the
+batch. Click :guilabel:`Confirm`, and the carrier is updated for all the selected records.
+
+.. figure:: dispatch/set-carrier.png
+   :alt: Set carrier.
+
+   The delivery method `Truck 1-MER-001` is set as the :guilabel:`Carrier` for two delivery orders.
+
+.. _inventory/shipping_receiving/create-batch:
+
+Create batch
+------------
+
+With the carrier set, begin adding orders to a batch or wave transfer by ticking the checkbox.
+
+.. note::
+  If a delivery order is already assigned to a batch transfer, assigning a batch transfer here does
+  **not** update it.
+
+Then, click the :icon:`fa-cog` :guilabel:`Actions` button, and click either :guilabel:`Add to batch`
+or :guilabel:`Add to wave`. In the pop-up window, ensure :guilabel:`Add to` is set to :guilabel:`a
+new [batch/wave] transfer`, then click :guilabel:`Confirm`.
+
+.. figure:: dispatch/add-to-wave.png
+   :alt: Example wave.
+
+   Delivery orders are selected to be grouped into a wave transfer.
+
+Alternative batch creation method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another place to create batches is by going to the :menuselection:`Inventory` app, and in the
+:guilabel:`Delivery Orders` card, click the :icon:`fa-ellipsis-v` :guilabel:`(three dots)` icon. In
+the resulting drop-down menu, click :guilabel:`Prepare batch`.
+
+.. note::
+   The :guilabel:`Transport Management` drop-down menu contains other tools for fleet management:
+
+   - :guilabel:`Manage Batches`: open list of batches
+   - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations
+   - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route
+   - :guilabel:`Calendar`: open hourly calendar view of scheduled operations
+   - :guilabel:`Statistics`: open pivot table of the batch transfers
+
+.. image:: dispatch/prepare-batch.png
+  :alt: Show prepare batch option from the new ... options.
```

Comment:
Image alt alignment and small alt tag change to be more descriptive. 

Also, I'd like to see this image cropped to just the delivery orders card, IMO there is no need to show the whole UI here :)

<img width="267" alt="Screenshot 2024-10-22 at 11 30 55 AM" src="https://github.com/user-attachments/assets/2b4ebb32-22e5-42c3-aab9-087b8613d790">

```suggestion
.. image:: dispatch/prepare-batch.png
   :alt: Show prepare batch option from the Transport Management drop-down menu.
```